src/sync/list.rs

```diff
@@ -3,6 +3,7 @@
 //! Michael.  High Performance Dynamic Lock-Free Hash Tables and List-Based Sets.  SPAA 2002.
 //! http://dl.acm.org/citation.cfm?id=564870.564881
 
+use std::mem::ManuallyDrop;
```

Comment:
What's the benefit of using `ManuallyDrop` here?

Reply:
Hmm, yeah - it's not necessary. I've removed `ManuallyDrop` from `sync::list`.